### PR TITLE
Properly denote a binary wheel

### LINKED ISSUE #2: Wheel is not marked as binary
Here's a patch from uwsgi:

https://github.com/unbit/uwsgi/commit/7ebe9575d317f011dc361b7aeebfc6f2a22150d3

### PR DESCRIPTION
**Pull request by asottile**
Resolves #2 

Before:
```
$ pip wheel .
$ ls *.whl
s6-2.2.2.0-py2.py3-none-any.whl
```

After:

```
$ pip wheel .
$ ls *.whl
s6-2.2.2.0-py2.py3-none-linux_x86_64.whl
```

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -139,7 +131,15 @@ def get_tag(self):
...
-    cmdclass=command_overrides,
+    cmdclass={
+        'sdist': sdist,
+        'bdist_wheel': bdist_wheel,
+        'fetch_sources': fetch_sources,
+        'build': build,
+        'build_s6': build_s6,
+        'install': install,
+        'install_cexe': install_cexe,
+    },
```
Comment (bukzor) [inline setup.py]: Unrelated diff obscures the change.

Comment (asottile) [inline setup.py]: It is related, before you were conditionally putting `bdist_wheel` into `cmdclass`, now it is unconditionally added (but `None` when there is no wheel support)

Comment (bukzor) [inline setup.py]: It's a matter of style, where the intent is a matter of functionality.
Changing both at once obscures the functional change.

**Comment by bukzor** [comment]
I'll merge this after putting it in the preceding style, then evaluate the style change separately.

File: setup.py (hunk 1 above)
Comment (asottile) [inline setup.py]: Do you want me to move classifiers, platforms, and name as well?

**Comment by asottile** [comment]
It's not a difference of style it's a functional change:

- I removed complexity (a function, 3 branches, and global mutation)

If you don't want the change. close this and implement it yourself.

**Comment by bukzor** [comment]
Closed via e6fa2488436
